source/detail/serialization/xlsx_producer.cpp

```diff
@@ -182,7 +182,10 @@ void xlsx_producer::begin_part(const path &part)
     end_part();
     current_part_streambuf_ = archive_->open(part);
     current_part_stream_.rdbuf(current_part_streambuf_.get());
-    current_part_serializer_.reset(new xml::serializer(current_part_stream_, part.string()));
+
+    auto xml_serializer = new xml::serializer(current_part_stream_, part.string(), 0);
+    xml_serializer->xml_decl("1.0", "UTF-8", "yes");
+    current_part_serializer_.reset(xml_serializer);
```

Comment:
original xml serializer does'nt write xml declaration, prettify xml (indent 2)
I saw my office 365 excel write xml declaration, no prettify xml, and I changed it.
(see inside example\17_xlsm.xlsm)